hosts.example

```diff
@@ -1,4 +1,4 @@
 [nodes]
-centos-1 IP=172.28.128.14
-centos-2 IP=172.28.128.15
-centos-3 IP=172.28.128.16
+centos-1 ip=172.28.128.14
```

Comment:
Why this needs to change? Nothing against it, but I'd like to understand

Reply:
etcd service refers to ['ip'] rather than IP. The dictionary lookup is case sensitive so we need the variables case to match

https://github.com/Arau/deploy-etcd-ansible/blob/4dc905430c0da62c52a4eff8901f184bdefaf8f2/roles/install_etcd/templates/etcd3.service#L21